tools/RAiDER/cli/raider.py

```diff
@@ -245,12 +246,27 @@ def calcDelays(iargs=None):
         # Grab the closest two times unless the user specifies 'nearest'
         # If the model time_delta is not specified then use 6
         # The two datetimes will be combined to a single file and processed
-        times = get_nearest_wmtimes(t, [model.dtime() if \
-                    model.dtime() is not None else 6][0]) if params['interpolate_time'] else [t]
```

Comment:
@bbuzz31 - this is the default behavior currently i.e. when the `interpolate_time` is not specified. What more are you asking?